lib/ts/controllers/s-expandable-control.ts

```diff
@@ -69,8 +69,8 @@
         // for non-checkable elements, the initial source of truth is the collapsed/expanded
         // state of the controlled element (unless the element doesn't exist)
         _isCollapsedForClickable() {
-            var cc = this.controlledCollapsible;
-            return cc ? !cc.classList.contains("is-expanded") : this.element.getAttribute("aria-expanded") === "false";
+            var cc = this.controlledCollapsibles;
+            return cc ? !cc.every(element => element.classList.contains("is-expanded")) : this.element.getAttribute("aria-expanded") === "false";
```

Comment:
```suggestion
            return cc.length > 0 ? !cc.every(element => element.classList.contains("is-expanded")) : this.element.getAttribute("aria-expanded") === "false";
```

`this.controlledCollapsibles` will never be null.  The equivalent to the before behavior is checking if we're empty.  That said, we never hit that condition before because we just threw within `controlledCollapsibles`